sphinx/source/docs/dev_guide.rst

```diff
@@ -168,6 +168,25 @@ to ``setup.py``:
 If you have any problems with the steps here, please contact the developers
 (see :ref:`contact`).
 
+Dependencies
+~~~~~~~~~~~~
+If you are working within a Conda environment, you will need to make sure you
+have the python requirements installed. You can install these via ``conda
+install`` or ``pip install`` for the packages referenced at
+:ref:`install_dependencies`.
```

Comment:
Mostly note to myself (and others), we probably need to update the dependencies list and add testing dependencies...